### DISCUSSION TITLE
Discoverability for Devs

### DISCUSSION THREAD

**Discussion by jacobdaddario** [joemasilotti/railsdevs.com#61] [Ideas]
As the list of developers on the site gets larger, we'll probably need some kind of way of creating discoverability for the site. For now the big index view is a little overwhelming.

**Comment by joemasilotti** [comment]
Definitely! Good idea. Some thoughts:

1. Sort by availability – sort by "Available now"
2. Sort by salary (coming soon)
3. Feature one dev per week (manual curation)
4. Highlight devs at the top (paid?)
5. Filter by `railsdevs.io` contributors
6. ...what else?

**Reply by jacobdaddario** [reply]
I definitely see doing paid promotions for devs as a reasonable source of revenue for the site. Additionally some filters themselves would be helpful on the index view.

Do you envision breaking out a UI that's separate from the index view, or would you want to keep it simple?

**Reply by joemasilotti** [reply]
The homepage of the website should probably have _some_ copy around what the site is about. Mostly for SEO. Then can lead into the featured/promoted devs. 

With a link for View all or something? But I'm open to ideas, I haven't thought about it much yet.

**Comment by joemasilotti** [comment]
With the homepage redesign, I think we can start iterating on this!

My first thought is to only show developers who are currently available on the homepage. That gets us down to ~15 folks.

Then, we can show a "See more" button at the end of the list to bring them to a filterable index. It also gives us room to create a CTA for developers to add their own profile.

**Reply by joemasilotti** [reply]
WIP PR incoming! #77

**Reply by jacobdaddario** [reply]
I like that idea as a start. I think it would fun to do some metric-based stuff too. For instance, we could feature new developers, sponsored developers, developers who've contributed to the project, etc.

**Reply by joemasilotti** [reply]
100%! There's a ton of directions for this and I'm excited to see what folks want to show off in this space.
